### ISSUE TITLE
Page of Request an appointment

### ISSUE THREAD

**Issue by minaleebtu** [astrojebus/GrandCottbusHospital#7]
Hi,

On request-appointment.html, should we just put the information from sample content?

Thank you.

**Comment by astrojebus** [comment]
Yes You can put the sample contents since it won’t use any forms
